### DISCUSSION TITLE
界面

### DISCUSSION THREAD

**Discussion by GuoXiangwei666** [community/community#64090] [Actions]
界面很好看 就是感觉对中国用户不太友好 github速度很慢

**Comment by christophercouch** [comment]
你好 @GuoXiangwei666，

谢谢你的反馈！我们知道在中国地区有时GitHub的访问速度可能不够理想。我们会努力改善这个问题，确保为所有用户提供更好的体验。

感谢你的支持！

**Comment by Akash1134** [comment]
Hi @GuoXiangwei666 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
